src/components/visualizations/chart/highcharts/test/customConfiguration.spec.ts

```diff
@@ -542,33 +542,39 @@ describe("getCustomizedConfiguration", () => {
     });
 
     describe("tooltip followPointer", () => {
-        it("should follow pointer for bar chart when data max is above axis max", () => {
-            const result = getCustomizedConfiguration({
-                ...chartOptions,
-                actions: { tooltip: true },
-                data: getData([{ y: 100 }, { y: 101 }]),
-                type: VisualizationTypes.COLUMN,
-                yAxisProps: {
-                    max: 50,
-                },
-            });
+        it.each([[VisualizationTypes.COLUMN], [VisualizationTypes.BAR], [VisualizationTypes.COMBO]])(
```

Comment:
should we reuse supportedTooltipFollowPointerChartTypes here?

Reply:
Fixed.